README.md

```diff
@@ -1 +1,165 @@
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 BudgetTrackr App <a name="about-project"></a>
+
+**BudgetTrackr App** This is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>localhost</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>PostgreSQL</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Keep track of all your expenses]**
+- **[Create your shopping categories and items with expenses]**
+
+  Live Video:[video](https://www.loom.com/share/9ff47a70c1ea4e3b966588a072caf8a0?sid=65515a75-d147-4349-91e2-a7d0ccf5c155)
+  Live App:[Live](https://budgettrack.onrender.com)
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- You'll need any code editor 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/ikoote1/BudgetTrackr.git
+  cd recipe-app
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install
+  rails db:create
+  rails db:migrate
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rails db:migrate RAILS_ENV=test
+  rspec
+```
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ikoote Rasuli**
+
+- GitHub: [@ikoote1](https://github.com/ikoote1)
+- LinkedIn: [Ikoote Rasuli](https://www.linkedin.com/in/ikooterasuli/)
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Improve UI design]**
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can give it a star!
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thanks Microverse for this learning opportunity.
+<p align="right">(<a href="#readme-top"></a>)</p>
```

Comment:
- [ ] [REQUIRED] Good job so far! The `README` contains an `"Acknowledgements"` section and similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) 🙏🏼